augur/application/cli/backend.py

```diff
@@ -162,6 +140,50 @@ def start(disable_collection, development, port):
             except RedisConnectionError:
                 pass
 
+def start_celery_worker_processes():
+
+    #Calculate process scaling based on how much memory is available on the system in bytes.
+    #Each celery process takes ~500MB or 500 * 1024^2 bytes
+
+    #Cap memory usage to 30% of total virtual memory
+    available_memory_in_bytes = psutil.virtual_memory().total * .3
+    available_memory_in_megabytes = available_memory_in_bytes / (1024 ** 2)
+    max_process_estimate = available_memory_in_megabytes // 500
+
+    #Get a subset of the maximum procesess available using a ratio, not exceeding a maximum value
+    def determine_worker_processes(ratio,maximum):
+        return min(round(max_process_estimate * ratio),maximum)
+
+    #2 processes are always reserved as a baseline.
+    scheduling_worker = f"celery -A augur.tasks.init.celery_app.celery_app worker -l info --concurrency=2 -n scheduling:{uuid.uuid4().hex}@%h -Q scheduling"
+
```

Comment:
I think 2 needs to be subtracted from `max_process_estimate` otherwise on small systems more than 30 percent of virtual memory will be used.

Reply:
That makes sense. I made the change.